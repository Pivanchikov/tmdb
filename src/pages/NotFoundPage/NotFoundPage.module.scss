@import '../../theme/colors.scss';

.container {
  min-height: calc(100vh - 128px);
  display: flex;
  justify-content: center;
  align-items: center;
}

.title {
  color: $darkBlue;
}

@media screen and (max-width: 745px) {
  .container {
    min-height: calc(100vh - 176px);
  }
}
