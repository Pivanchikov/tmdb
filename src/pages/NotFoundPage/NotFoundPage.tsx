import classes from './NotFoundPage.module.scss'

export const NotFoundPage = () => (
  <div className={classes.container}>
    <h1 className={classes.title}>Error 404. Page not found</h1>
  </div>
)
