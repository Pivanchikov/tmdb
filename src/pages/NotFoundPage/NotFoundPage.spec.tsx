import { render } from '@testing-library/react'

import { NotFoundPage } from './NotFoundPage'

describe('render Not Found Page', () => {
  const setup = () => render(<NotFoundPage />)

  it('matches snapshot', () => {
    const { asFragment } = setup()

    expect(asFragment()).toMatchSnapshot()
  })
})
