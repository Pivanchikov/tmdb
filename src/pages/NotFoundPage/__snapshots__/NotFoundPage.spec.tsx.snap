// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Not Found Page matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Error 404. Page not found
    </h1>
  </div>
</DocumentFragment>
`;
